chromium_src/ui/shell_dialogs/execute_select_file_win.cc

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "ui/shell_dialogs/execute_select_file_win.h"
+
+#include <shlobjidl.h>
+
+// Disable FOS_FORCEFILESYSTEM.  If we set FOS_FORCEFILESYSTEM, and
+// you type a URL into the file picker, Windows will download that URL
+// and return a pathname to the downloaded file, with network traffic
+// beyond our control.
+#undef FOS_FORCEFILESYSTEM
+#define FOS_FORCEFILESYSTEM 0
```

Comment:
in the original report, reporter mentioned unsetting `FOS_FORCEFILESYSTEM` is insufficient, we also need to set `FOS_SUPPORTSTREAMABLEITEMS`. How about we define `FOS_FORCEFILESYSTEM` as `FOS_SUPPORTSTREAMABLEITEMS` for this file?

Reply:
Oops, so there is, I missed that.  Sounds reasonable.  We may also have to worry about https://bugs.chromium.org/p/chromium/issues/detail?id=939985.